### PR TITLE
EN-19514: Remove region coding computation

### PR DESCRIPTION
**Pull request by aerust**
Remove region coding computation paths from Soda-fountain.

Leave the /compute endpoint and add some logging which I think is
unused until we are confident that it is safe to get rid of.

### CODE REVIEW

File: soda-fountain-lib/src/main/scala/com/socrata/soda/server/resources/Dataset.scala [hunk 1]
```diff
@@ -8,7 +8,7 @@ import com.socrata.http.server.util.RequestId
 import com.socrata.soda.server._
 import com.socrata.soda.server.copy.Stage
 import com.socrata.soda.server.responses._
-import com.socrata.soda.server.highlevel.DatasetDAO.{NonExistentColumn, DatasetAlreadyExists}
+//import com.socrata.soda.server.highlevel.DatasetDAO.{NonExistentColumn, DatasetAlreadyExists}
```
Comment (chitang) [inline soda-fountain-lib/src/main/scala/com/socrata/soda/server/resources/Dataset.scala]: You don't mean to check this in?

File: soda-fountain-lib/src/main/scala/com/socrata/soda/server/resources/Dataset.scala [hunk 2]
```diff
@@ -8,7 +8,7 @@ import com.socrata.http.server.util.RequestId
...
 import com.socrata.soda.server.highlevel._
```
Comment (aerust) [inline soda-fountain-lib/src/main/scala/com/socrata/soda/server/resources/Dataset.scala]: oops I'll delete this

**Comment by chitang** [comment]
Compute isn't really doing anything even though you leave the endpoint there?

**Comment by chitang** [comment]
🚢 red is good

**Pull request merged**